src/screens/SettingsSheet/components/GoogleAccountSection.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect, useState } from 'react';
+import {
+  getGoogleAccountUserData,
+  GoogleDriveUserData,
+  logoutFromGoogleDrive,
+} from '@/handlers/cloudBackup';
+import ImageAvatar from '@/components/contacts/ImageAvatar';
+import { showActionSheetWithOptions } from '@/utils';
+import * as i18n from '@/languages';
+import {
+  clearAllWalletsBackupStatus,
+  updateWalletBackupStatusesBasedOnCloudUserData,
+} from '@/redux/wallets';
+import { useDispatch } from 'react-redux';
+import Menu from './Menu';
+import MenuItem from './MenuItem';
+import { logger, RainbowError } from '@/logger';
+
+export const GoogleAccountSection: React.FC = () => {
+  const dispatch = useDispatch();
+  const [accountDetails, setAccountDetails] = useState<
+    GoogleDriveUserData | undefined
+  >(undefined);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    getGoogleAccountUserData()
+      .then(accountDetails => {
+        setAccountDetails(accountDetails ?? undefined);
+      })
+      .catch(error => {
+        logger.error(
+          new RainbowError(
+            `Something went wrong when trying to fetch google account data to display in Backups Section, error: ${error?.toString()}`
+          )
```

Comment:
Nit: best practice with Sentry is to use short error messages and add metadata as needed. Interpolating the error message means that this single thrown error may appear as separate issues in Sentry.

Something like this would be preferable I think:
```suggestion
          new RainbowError(
            `SettingsSheet error, failed to fetch Gogogle account data`
          ),
          { error: (error as Error).message }
```

Reply:
You don't need to actually use this, just nice to have the diff